critiquebrainz/frontend/views/artist.py

```diff
@@ -87,15 +94,9 @@ def _squash_duplicated_members(members):
 
 
 def _get_period(member):
```

Comment:
Please document what the input and output of this function is and why it's necessary.